### PR TITLE
Sidecar fixes

### PR DESCRIPTION
**Pull request by rocky341**
Hi @t92549 @GCHQDeveloper314 

Some small fixes which are needed in order for the sidecars to work without issue

### CODE REVIEW

File: gaffer-as-a-service/gaas-rest/pom.xml [hunk 1]
```diff
@@ -251,8 +237,11 @@
             <version>1.12.8</version>
             <scope>test</scope>
         </dependency>
-
-
+        <dependency>
+            <groupId>ch.qos.reload4j</groupId>
+            <artifactId>reload4j</artifactId>
+            <version>1.2.20</version>
+        </dependency>
```
Comment (GCHQDeveloper314) [inline gaffer-as-a-service/gaas-rest/pom.xml:244]: Why is the version specified? It should be inherited from the Gaffer parent POM.

File: gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/handlers/DeploymentHandler.java [hunk 2]
```diff
@@ -54,7 +53,7 @@
 public class DeploymentHandler {
 
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(DeploymentHandler.class);
+    private static final Logger LOGGER = Logger.getLogger(DeploymentHandler.class);
```
Comment (GCHQDeveloper314) [inline gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/handlers/DeploymentHandler.java:56]: Why is this being changed from using SLF4J? SLF4J should be used unless a feature is required from Reload4J which does not exist in SLF4J. I don't think this change is required.

**Review by GCHQDeveloper314 — CHANGES_REQUESTED** [review]

File: gaffer-as-a-service/gaas-rest/pom.xml (hunk 1 above)
Comment (rocky341) [reply gaffer-as-a-service/gaas-rest/pom.xml:244]: Removed

File: gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/handlers/DeploymentHandler.java (hunk 2 above)
Comment (rocky341) [reply gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/handlers/DeploymentHandler.java:56]: Reverted

**Review by t92549 — APPROVED** [review]

**Review by GCHQDeveloper314 — APPROVED** [review]

**Pull request merged**